docker/Dockerfile

```diff
@@ -10,7 +10,9 @@ LABEL org.opencontainers.image.authors="oss@sentry.io"
 
 # Sane defaults for pip
 ENV PIP_NO_CACHE_DIR=off \
-    PIP_DISABLE_PIP_VERSION_CHECK=1
+    PIP_DISABLE_PIP_VERSION_CHECK=1 \
+    # Webpack needs to know this is CI/headless env
+    CI=1
```

Comment:
Should we use a different env var?

Reply:
Ideally we wouldn't need to set anything related to this inside Docker specifically. See my commend below. I added this to see if my theory is correct and then to unblock @priscilawebdev for landing this patch.

Reply:
I think there was some mixed boolean logic so when you apply my suggestions, you should be able to revert this change.